app/kubernetes/views/configure/configureController.js

```diff
@@ -224,8 +224,6 @@ class KubernetesConfigureController {
       await Promise.all(storagePromises);
 
       this.Notifications.success('Success', 'Configuration successfully applied');
-
-      this.$state.go('portainer.home');
```

Comment:
Now that you don't get directed away from the page (a nice behaviour), is there any weird behaviour when you try to change setup settings after clicking save?

Reply:
Doesn't seem to be any strange behavior. I tried creating a brand new cluster, going into settings, disabling metrics, saving, enabling it, saving and nothing seemed out of the ordinary.

Reply:
Might not hurt to mention this on the ticket though so QA can mess with it further and ensure there's no weird edge cases.

Reply:
Went with your suggestion, thanks!